istioctl/cmd/istioctl/rbac.go

```diff
@@ -16,14 +16,153 @@ package main
 
 import (
 	"fmt"
+	"io"
+	"strings"
 
+	"github.com/ghodss/yaml"
+	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
 
 	rbacproto "istio.io/api/rbac/v1alpha1"
 	"istio.io/istio/mixer/adapter/rbac"
 	"istio.io/istio/pilot/pkg/model"
 )
 
+// list provides a command named list that allows user to query information under current
+// Istio RBAC policies.
+func list() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Query information under current Istio RBAC policies.",
+		Long: `
+This command lets you query information under current Istio RBAC policies.
+You can list all permissions a subject has, or all members that are allowed to access a service`,
+	}
+
+	cmd.AddCommand(permissions())
+	cmd.AddCommand(members())
+
+	return cmd
+}
+
+// accessRules allows user to list AccessRules under current Istio RBAC policies for a given subject.
+func permissions() *cobra.Command {
+	subject := rbac.SubjectArgs{}
+	action := rbac.ActionArgs{}
+
+	cmd := &cobra.Command{
+		Use:   "permissions",
+		Short: "List AccessRules a subject has under current Istio RBAC policies",
```

Comment:
The doc in `Long` used the term `permissions`, the doc in `Short` used the term `AccessRules`, maybe they should be unified?

Reply:
Fixed